chanzuckerberg/idseq-web
# [Mapping] Start map_playground endpoint and MapPlayground components

### Description
- Starts a locations/map_playground endpoint that will return currently viewable values for MetadataField collection_location. Eventually there will be new Location objects to use instead.
- Endpoint tests in locations_controller_test.rb.
- Starts MapPlayground and BaseMap React components.
- Includes setup for react-map-gl and tiles from MapTiler.
- NPM changes were produced by running `npm install react-map-gl`

### Demo
- Will look like this (but more zoomed in)
![Screen Shot 2019-04-15 at 5 58 07 PM](https://user-images.githubusercontent.com/5652739/56174884-9119a700-5fa9-11e9-9c4d-a9aeeb1a069d.png)

### Test and Reproduce
- Pull the branch, retrieve MAPTILER_API_KEY from Parameter Store, start locally with `MAPTILER_API_KEY=key_here docker-compose up web`
- Visit http://localhost:3000/locations/map_playground
- See map displayed and explore
